shippable.yml

```diff
@@ -35,6 +35,7 @@ build:
     # install pytest-xdist to leverage a second core
     # for unit tests
     - pip install pytest-xdist
+    - if [ $(python -c 'import sys; print(sys.version[0])') == "3" ]; then pip install pickle5; fi
```

Comment:
If the python version is 3.5 or 3.8 this command will fail.

The test should pass if pickle5 is not there. I think this can be removed now.